cookbooks/aws-parallelcluster-install/resources/mysql_client/mysql_client.rb

```diff
@@ -38,3 +39,47 @@
   # Add MySQL source file
   action_create_source_link
 end
+
+action_class do
+  def package_platform
+    if arm_instance?
+      value_for_platform(
+        'default' => "el/7/aarch64"
+      )
+    else
+      value_for_platform(
+        'default' => "el/7/x86_64",
+        'ubuntu' => {
+          '20.04' => "ubuntu/20.04/x86_64",
+          '18.04' => "ubuntu/18.04/x86_64",
+        }
+      )
+    end
+  end
+
+  def expected_version
+    if arm_instance?
+      "8.0.31"
```

Comment:
If we want to change the version of mysql we need to change this value in multiple places: expected_version, package_version and package_source_version

Can we use it the `package_source_version`  variable?

Reply:
This was and it is used by the mysql-validate. I kept it since I assumed we needed a different value for checking.

We can remove it in a future iteration if we consider it a duplicate.